en/orm/database-basics.rst

```diff
@@ -648,7 +648,7 @@ value object and into SQL expressions::
     {
         public function toPHP($value, DriverInterface $d)
         {
-            return Point::parse($value);
+            return null === $value ? null : Point::parse($value);
```

Comment:
Can we please not use yoda

Reply:
sorry, VS Code with intellephense "schooled" me to do it this way around :D

so would 
```
return $value === null ? null : Point::parse($value);
```

be ok or rather

```
return is_null($value) ? null : Point::parse($value);
```

Reply:
The former is fine :)